api/python/cellxgene_census/src/cellxgene_census/experimental/pp/_online.py

```diff
@@ -58,14 +71,19 @@ def finalize(
 
         # accum all batches using Chan's
         all_u, all_M2 = _mbomv_combine_batches(self.n_batches, self.n_samples, self.u, self.M2)
+
         with np.errstate(divide="ignore"):
-            all_var = all_M2 / max(0, (self.n_samples.sum() - self.ddof))
+            if self.exclude_zeros:
+                all_var = all_M2 / np.maximum(0, self.n - self.ddof)
+                all_var = all_var[0]
+            else:
+                all_var = all_M2 / max(0, (self.n_samples.sum() - self.ddof))
```

Comment:
i would be consistent on use of max vs np.maximum

I'm not sure which is right, but I assume that np.maximum was introduced for a reason?

Reply:
`np.maximum` does an element-wise maximum on the array here, while the latter is a scalar.

Reply:
Ah ok, good point. We can go for `np.maximum` then. I'll make the change, thanks!